scene/gui/code_edit.cpp

```diff
@@ -1626,6 +1666,33 @@ void CodeEdit::unfold_all_lines() {
 	_unhide_all_lines();
 }
 
+void CodeEdit::create_code_region() {
```

Comment:
This looks like it needs support adding for multiple carets

Reply:
Not sure to understand what you mean here.

If multiselection is active, right mouse click unselect all the selections and so we do not have access to the "Create region" menu item.

So maybe we just need to early return if multiselection is active so that the keyboard shortcut has no effect too.

Or did you mean an other thing ?